### DISCUSSION TITLE
Zotero annotiations get written down but also get deleted right after

### DISCUSSION THREAD

**Discussion by WardUlenaers** [BryanJenksCommunity/FAQ#178] [Q&A] [answered]
I followed your video on how you use the Zotero Integration plugin and it works for the most part.
The problem I'm having is that all my annotations are being collected and written down in my obsidian but are also deleted right after that.

I'm doing this on a Windows 10 machine and below is a video of what happens.

https://github.com/BryanJenksCommunity/FAQ/assets/70289175/cc3b8bab-8ae4-4486-95d5-6c73f5897b39

**Comment by tallguyjenks** [comment]
Thank you for the video! that makes it much easier to assist!

Questions:

- are you using my template?
  - if so, did you change ANYTHING or is it exactly as is?
- What does your configuration settings look like for the plugin?

**Reply by WardUlenaers** [reply]
Normally i copied everything except the backlink to the annotation because I didn't know how you did that. Here is the code itself:
```
{% for annotation in annotations -%} 
    {%- if annotation.annotatedText -%} 
	    - <mark class="hltr-{{annotation.colorCategory | lower}}">"{{annotation.annotatedText}}"</mark> 
    {%- endif %} 
    {%- if annotation.imageRelativePath -%}
    ![[{{annotation.imageRelativePath}}]]
    {%- endif %} 
{% if annotation.comment %} 
	- {{annotation.comment}}
{% endif %} 
{% endfor -%}
```

And here are the settings i used in the Zotero integration plugin:

![1](https://github.com/BryanJenksCommunity/FAQ/assets/70289175/86aaacff-c00b-4e2b-a488-14f93900ced5)
![2](https://github.com/BryanJenksCommunity/FAQ/assets/70289175/fba48e6b-689d-4ec6-9b82-86749423ddc4)
![3](https://github.com/BryanJenksCommunity/FAQ/assets/70289175/7b71f509-1268-4afc-b42d-1eb3b93cdf95)

**Reply by tallguyjenks** [reply]
is this still an issue? i recently ran into this in a sandbox vault and wasn't sure why it was happening either.

**Reply by WardUlenaers** [reply, marked as answer]
Sorry for answering this late, I had forgotten about this post.
This isn't an issue anymore for me. I don't do anything different now but the issue is somehow resolved and it works again.
Will let you know if it happens again.
